### PR TITLE
feat: added responsive image component

### PR DESCRIPTION
**Pull request by GuptaEesh**
2 types are added:

>Responsive Image
Responsive Circular Image

images

### CODE REVIEW

File: components/images.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.eg-img-container {
+    background-color: var(--white);
+    border: 2px solid var(--secondary-200);
+    border-radius: 5px;
+    display: inline-block;
+    padding: 8px;
+}
+
+.img-responsive {
+    border-radius: 5px;
```
Comment (saadahmadhere) [inline components/images.css]: You can use the values in the multiples of 4 or 8. Not necessary, just a suggestion.

**Review by saadahmadhere — APPROVED** [review]
Looks nice and written well. Good to merge

File: components/images.css (hunk 1 above)
Comment (GuptaEesh) [reply components/images.css]: Done✅

**Review by raghav2110 — APPROVED** [review]
Class name are on point. Code is clean and readable. Go work done.

**Comment by GuptaEesh** [comment]
Got 2 reviews. Took the suggestion and applied it. Merging now.

**Pull request merged**